Makefile

```diff
@@ -11,6 +11,9 @@ install:
 clean:
 	rm -rf cellxgene_schema_cli/build cellxgene_schema_cli/dist cellxgene_schema_cli/cellxgene_schema.egg-info
 
+mypy:
+	mypy --config-file pyproject.toml
+
```

Comment:
seems like fmt/lint/mypy could all just be folded together into something like `check` or even `pre-commit`. Out-of-scope fort the PR, perhaps, but can't flake8 also be run by pre-commit?

Reply:
yeah that seems reasonable, i can consolidate